scripts/deblur

```diff
@@ -647,6 +650,8 @@ def workflow(seqs_fp, output_dir, pos_ref_fp, pos_ref_db_fp,
         for f in files_to_remove:
             remove(f)
         rmtree(working_dir)
+        if created_split_dir is not None:
```

Comment:
should not be undefined in current implementation but agree could complicate things later if logic is changes. moved the initialization closer to beginning of pipeline.